litellm/integrations/langfuse.py

```diff
@@ -250,30 +250,57 @@ def _log_langfuse_v2(
             print_verbose(f"Langfuse Layer Logging - logging to langfuse v2 ")
 
             if supports_tags:
-                metadata_tags = metadata.get("tags", [])
+                metadata_tags = metadata.pop("tags", [])
                 tags = metadata_tags
 
-            trace_name = metadata.get("trace_name", None)
+            trace_name = metadata.pop("trace_name", None)
             if trace_name is None:
                 # just log `litellm-{call_type}` as the trace name
                 trace_name = f"litellm-{kwargs.get('call_type', 'completion')}"
 
             trace_params = {
                 "name": trace_name,
                 "input": input,
-                "user_id": metadata.get("trace_user_id", user_id),
-                "id": metadata.get("trace_id", None),
-                "session_id": metadata.get("session_id", None),
+                "user_id": metadata.pop("trace_user_id", user_id),
```

Comment:
why change the `get` to `pop`? @Manouchehri

Reply:
pop is dangerous here - as kwargs are used by multiple integrations in the for-loops